src/libs/globals.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash/fp'
+import { forwardRef } from 'react'
+import { h } from 'react-hyperscript-helpers'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+
+
+const globals = Utils.atom(JSON.parse(sessionStorage['globals'] || '{}'))
```

Comment:
I like the approach of using `sessionStorage`, but I don't think it's a good idea to introduce an open-ended 'globals' store at this point. We have some pretty good tooling around atoms that we can leverage for one-off cases like this.